atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -189,6 +254,12 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
         self.build_kwargs = build_kwargs
         self.osbs_client_config = osbs_client_config
         self.config_kwargs = config_kwargs or {}
+        self.find_cluster_retry_delay = find_cluster_retry_delay
```

Comment:
This is redundant and can be removed.

Reply:
Right now it's designed to be configurable because we haven't figured out exactly how we want it to work in production. We can always just set them to the same value if that's the desired behavior.

Reply:
I'm fine either way.